openml/extensions/sklearn/run_functions.py

```diff
@@ -0,0 +1,453 @@
+from collections import OrderedDict
```

Comment:
since this file is completely new anyway, I would go ahead and copy the functions in the `SklearnExtension` class already. This saves us time later for the refactoring  (in particular for reviewing, as the diff will be smaller).